### PR TITLE
Incorporating Browser Page to the Default App

### PR DESCRIPTION
**Pull request by sukusuma**
a. Browser Page
b. Connect to Browser Page on Open WiFi Network Connections
c. TopBar User Control and applied across Pages
Minor fixes

### CODE REVIEW

File: IoTCoreDefaultApp/IoTCoreDefaultApp/Views/WebBrowserPage.xaml.cs [hunk 1]
```diff
@@ -277,7 +277,7 @@ void ContentView_LongRunningScriptDetected(WebView sender, WebViewLongRunningScr
             if (args.ExecutionTime > new TimeSpan(0, 0, 10))
             {
                 args.StopPageScriptExecution = true;
-                WebAddressText.QueryIcon = new SymbolIcon(Symbol.Clear);
+                //WebAddressText.QueryIcon = new SymbolIcon(Symbol.Clear);
```
Comment (paulmon) [inline IoTCoreDefaultApp/IoTCoreDefaultApp/Views/WebBrowserPage.xaml.cs]: Are these comments dead code?  If yes, please delete

File: IoTCoreDefaultApp/IoTCoreDefaultApp/Views/Settings.xaml.cs [hunk 2]
```diff
@@ -456,51 +457,81 @@ private void SettingsChoice_ItemClick(object sender, ItemClickEventArgs e)
             // Language, Network, or Bluetooth settings etc.
             SwitchToSelectedSettingsAsync(item.Name);
         }
-        private void CollapseGridViews()
+        private void CollapseGridViews(string iName)
```
Comment (paulmon) [inline IoTCoreDefaultApp/IoTCoreDefaultApp/Views/Settings.xaml.cs]: Consider renaming CollapseGridViews to ShowOneGridView or SelectGridView.  Or maybe CollapseGridViewsExceptOne?  Is there a reason this method is separate from SwitchToSelectedSettingsAsync?  They seem pretty tightly couple now.

**Comment by paulmon** [comment]
It seems like the merge in the first change should be a rebase rather than merge.

File: IoTCoreDefaultApp/IoTCoreDefaultApp/Views/Settings.xaml.cs (hunk 2 above)
Comment (sukusuma) [reply IoTCoreDefaultApp/IoTCoreDefaultApp/Views/Settings.xaml.cs]: Started with other logic and later merged.  will merge into one function

File: IoTCoreDefaultApp/IoTCoreDefaultApp/Views/WebBrowserPage.xaml.cs (hunk 1 above)
Comment (sukusuma) [reply IoTCoreDefaultApp/IoTCoreDefaultApp/Views/WebBrowserPage.xaml.cs]: Deleting all commented code out.

File: IoTCoreDefaultApp/IoTCoreDefaultApp/Utils/Constants.cs [hunk 3]
```diff
@@ -14,6 +14,11 @@ public static class Constants
             "GetConnected",
             "GetCoding"
         };
+
+        public static string WODUrl = "WindowsOnDevices.com";
```
Comment (rachitb777) [inline IoTCoreDefaultApp/IoTCoreDefaultApp/Utils/Constants.cs]: Make them consts as well.

File: IoTCoreDefaultApp/IoTCoreDefaultApp/Views/CommandLinePage.xaml [hunk 4]
```diff
@@ -80,7 +82,7 @@
                     <ColumnDefinition Width="Auto"/>
                 </Grid.ColumnDefinitions>
 
-                <TextBox Grid.Column="0" x:Name="CommandLine" PlaceholderText="{Binding [CommandLinePlaceholderText]}" Background="Transparent" BorderThickness="2" BorderBrush="Transparent" FontFamily="Lucida Console" FontSize="14" Padding="6 6" KeyUp="CommandLine_KeyUp" Height="24" HorizontalAlignment="Stretch" VerticalAlignment="Stretch" IsSpellCheckEnabled="False"/>
+                <TextBox Grid.Column="0" x:Name="CommandLine" PlaceholderText="{Binding [CommandLinePlaceholderText]}" Background="Transparent" BorderThickness="2" BorderBrush="Transparent" FontFamily="Lucida Console" FontSize="14" Padding="6 6" KeyUp="CommandLine_KeyUp" Height="24" HorizontalAlignment="Stretch" VerticalAlignment="Stretch" IsSpellCheckEnabled="False"  />
```
Comment (rachitb777) [inline IoTCoreDefaultApp/IoTCoreDefaultApp/Views/CommandLinePage.xaml]: No change here. Remove the extra space.

File: IoTCoreDefaultApp/IoTCoreDefaultApp/Views/MainPage.xaml [hunk 5]
```diff
@@ -156,10 +92,11 @@
                                 <TextBlock Text="{Binding [VisitText]}" Style="{StaticResource SubtitleTextBlockStyle}"/>
                             </InlineUIContainer>
                             <InlineUIContainer>
-                                <TextBlock Text="{Binding [WindowsOnDevicesUrlText]}" Style="{StaticResource SubtitleTextBlockStyle}" Foreground="{StaticResource AccentBrush}"/>
-                            </InlineUIContainer>
-                            <InlineUIContainer>
-                                <TextBlock Text="{Binding [StartDevelopingText]}" Style="{StaticResource SubtitleTextBlockStyle}"/>
+                                <TextBlock Style="{StaticResource SubtitleTextBlockStyle}" Foreground="{StaticResource AccentBrush}">
+                                    <Hyperlink NavigateUri="{Binding [WindowsOnDevicesUrlText]}" Click="WindowsOnDevices_Click" ToolTipService.ToolTip="{Binding [WindowsOnDevicesUrlText]}" >
+                                        Windows 10 IoT Core
```
Comment (rachitb777) [inline IoTCoreDefaultApp/IoTCoreDefaultApp/Views/MainPage.xaml]: Add this to resex

File: IoTCoreDefaultApp/IoTCoreDefaultApp/Views/Settings.xaml.cs [hunk 6]
```diff
@@ -29,7 +29,7 @@ namespace IoTCoreDefaultApp
     public sealed partial class Settings : Page
     {
         private LanguageManager languageManager;
-        private UIElement visibleContent;
+        //private UIElement visibleContent;
```
Comment (rachitb777) [inline IoTCoreDefaultApp/IoTCoreDefaultApp/Views/Settings.xaml.cs]: Remove if this is not used.

**Comment by rachitb777** [comment]
Looks good. Added a few comments. Also please rebase the changes on ms-iot/develop

File: IoTCoreDefaultApp/IoTCoreDefaultApp/Strings/en-US/Resources.resw [hunk 7]
```diff
@@ -166,6 +166,7 @@
   </data>
   <data name="NetworkConnectingMessageText" xml:space="preserve">
     <value>Checking network requirements</value>
+    <comment>Message disply during wifi network connect click</comment>
```
Comment (rachitb777) [inline IoTCoreDefaultApp/IoTCoreDefaultApp/Strings/en-US/Resources.resw]: display

**Comment by rachitb777** [comment]
👍

**Pull request merged**